tabrepo/models/catboost/generate_alt.py

```diff
@@ -0,0 +1,42 @@
+from ConfigSpace import ConfigurationSpace, Float, Categorical, Integer
```

Comment:
ConfigSpace I remember didn't work on Windows a long time ago. Just noting that it caused some dependency issues with AutoGluon in the past, and I'd prefer to use something lighter weight if we don't end up needing the advanced logic that exists in ConfigSpace.